build.gradle

```diff
@@ -3,10 +3,24 @@ description = "Apereo uPortal $version"
 // Build Scans provide detailed information about many aspects of your
 // build right in your browser.  (https://scans.gradle.com/plugin)
 apply plugin: 'com.gradle.build-scan'
+apply plugin: 'java'
 
 // Adds support for Node.js scripts
 apply plugin: 'com.moowork.node'
 
+def props = new Properties()
+file("build.properties").withInputStream { props.load(it) }
+
+task copySampleProperties(type: Copy) {
```

Comment:
@stalele If `build.properties` does not exist, Gradle will fail before it can copy it.
This is what is currently happening on CI.
https://travis-ci.org/Jasig/uPortal/builds/233427091#L187-L192